UnitedIncome/components
# [APP-5100] TableTile

### Description
> Please provide a brief description of your changes below.

 Addressed vertical alignment of text for table tile stories

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

Regarding the TableTile stories ensure that the text displayed in each cell is vertically aligned.

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

### Comments
> Any additional comments that will aid with the review process.

- [APP-5100](https://unitedincome.atlassian.net/browse/app-5100)
